server/cluster/cluster.go

```diff
@@ -438,13 +438,25 @@ func (c *Cluster) GetClusterState() storage.ClusterState {
 	return c.topologyManager.GetClusterState()
 }
 
+func (c *Cluster) GetClusterView() storage.ClusterView {
+	return c.topologyManager.GetClusterView()
+}
+
 func (c *Cluster) UpdateClusterView(ctx context.Context, state storage.ClusterState, shardNodes []storage.ShardNode) error {
 	if err := c.topologyManager.UpdateClusterView(ctx, state, shardNodes); err != nil {
 		return errors.WithMessage(err, "update cluster view")
 	}
 	return nil
 }
 
+func (c *Cluster) GetShardViews() []storage.ShardView {
```

Comment:
Why not name it as `RangeShardViews` too?

Reply:
Remove it, and add `GetTopology` in `GetClusterSnapshot`.